apps/zui/src/electron/run-main/before-boot.ts

```diff
@@ -6,16 +6,18 @@ import {MainArgs} from "./args"
 import {setLogLevel} from "../set-log-level"
 import {runMigrations} from "./run-migrations"
 
+app.disableHardwareAcceleration()
+
 export async function beforeBoot(
   args: Partial<MainArgs>
 ): Promise<string | null> {
   // Disable security warnings
-  process.env.ELECTRON_DISABLE_SECURITY_WARNINGS = "true"
+  // process.env.ELECTRON_DISABLE_SECURITY_WARNINGS = "true"
   // Setup app paths, this must be first
   appPathSetup()
   setLogLevel()
   // Disable for certain platforms
-  app.disableHardwareAcceleration()
```

Comment:
This rang a bell when I saw it and so I did a search and just dug up #1248 where we disabled it to fix some specific issues. If it's absolutely necessary we revert this we could invest some time to try and reproduce the kinds of issues we'd fixed with this in the past and see if they've been chased away for some reason. But given how many other unsolved problems we've got on our list I'd rather avoid bringing back old ones if we can.